package.json

```diff
@@ -20,11 +20,11 @@
   ],
   "license": "Apache-2.0",
   "peerDependencies": {
-    "@openfeature/js-sdk": "^1.0.0",
```

Comment:
The OpenFeature SDK was renamed to make it clear that their is a client and server version.